akka-contrib/src/main/scala/akka/contrib/pattern/TransparentExponentialBackoffSupervisor.scala

```diff
@@ -29,9 +29,9 @@ object TransparentExponentialBackoffSupervisor {
    *       maxBackoff,
    *       (random_delay_factor * calculated_backoff) + calculated_backoff)
    *   }}}
-   * @param decider an [[akka.actor.SupervisorStrategy.Decider]] to specify how the supervisor
+   * @param decider an `Decider` to specify how the supervisor
```

Comment:
`$` did work for `Restart`, because there's a restart class. 
It did not work with `Decider`. Because `Decider` is just a type, it is converted into Java as is - `scala.PartialFunction<java.lang.Throwable, akka.actor.SupervisorStrategy.Directive> decider` so it cannot be found by javadoc.